src/ontology/mondo-edit.obo

```diff
@@ -621201,6 +621194,15 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 property_value: http://purl.org/dc/terms/creator "https://orcid.org/0000-0002-7638-4659" xsd:string
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/9596" xsd:anyURI
 
+[Term]
+id: MONDO:1060180
+name: brain stem glioblastoma
+def: "A glioblastoma localized in the brain stem." [https://orcid.org/0000-0001-6677-8489, NCIT:C5097]
+xref: NCIT:C5097 {source="MONDO:equivalentTo"}
+is_a: MONDO:0002501 {source="NCIT:C5097", source="https://orcid.org/0000-0001-6677-8489"} ! brain glioblastoma
```

Comment:
@MeeSiing you can also add a "(disease has location) some %s " annotation (and equivalent definition), following this pattern:
https://github.com/monarch-initiative/mondo/blob/master/src/patterns/dosdp-patterns/neoplasm.yaml